File: api/DTOs/Admin/CVRefDto.cs
using api.DTOs.Process;

namespace api.DTOs.Admin
{
    public class CVRefDto
    {
        public int Id { get; set; }
        public int CVRefId { get; set; }
        public bool Checked {get; set;}
        public int CustomerId { get; set; }
        public int CandidateId { get; set; }
        public int ApplicationNo { get; set; }
        public string CandidateName { get; set; }
        public string CustomerName { get; set; }
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateOnly OrderDate {get; set;}
        public int OrderItemId { get; set; }    
        public int SrNo {get; set;}
        public string ProfessionName { get; set; }
        public string CategoryRef { get; set; }
        public string PPNo {get; set;}
        public DateOnly ReferredOn { get; set; }
        public string RefStatus { get; set; }
        public string SelectionStatus { get; set; }
    }
}

File: client/src/app/_dtos/admin/cvRefDto.ts
import { ICVRefDeployDto } from "./cvRefDeployDto";
import { IDeployDto } from "./cvReferredDto";

export interface ICVRefDto
{
     id: number;

     checked: boolean;
     customerId: number;
     customerName: string;
     candidateId: number;
     orderId: number;
     orderNo: number;
     orderDate: Date;
     orderItemId: number;
     srNo: number;
     refStatus: string;
     selectionStatus: string;

     professionName: string;
     cvRefId: number;
     categoryRef: string;
     applicationNo: number;
     candidateName: string;
     referredOn: Date;
     selectedOn: Date;
     
}

export class CVRefDto implements ICVRefDto
{
     id = 0;
     checked = false;
     customerId = 0;
     customerName = '';
     candidateId = 0;
     orderId = 0;
     orderNo = 0;
     orderDate = new Date();
     orderItemId = 0;
     srNo = 0;
     refStatus = '';
     selectionStatus = '';
     professionName = '';
     
     cvRefId = 0;
     categoryRef = '';
     applicationNo = 0;
     candidateName = '';
     referredOn = new Date();
     selectedOn = new Date();

}

